src/components/attributes/Attributes.js

```diff
@@ -70,18 +69,43 @@ class AddComponent extends React.Component {
 }
 
 const Attributes = props => {
-  var entity = props.entity;
-  var components = entity ? props.entity.components : {};
+  const entity = props.entity;
+  const components = entity ? entity.components : {};
+  const defaultComponents = Object.keys(components).filter(function (key) {
+    return DEFAULT_COMPONENTS.indexOf(key) === -1;
+  }).sort().map(function (key) {
+    return <Component entity={entity} key={key} name={key} component={components[key]}/>;
+  });
+
   return (
-    <div className="attributes">
+    <div className='attributes'>
       <CommonComponents entity={entity}/>
       <AddComponent entity={entity}/>
-      {
-         Object.keys(components).filter(function(key){return ['visible','position','scale','rotation'].indexOf(key)==-1;}).sort().map(function(key) {
-            return <Component entity={entity} key={key} name={key} component={components[key]}/>
-         })
-      }
+      {defaultComponents}
     </div>
   );
 };
+Attributes.PropTypes = {
+  entity: React.PropTypes.element.isRequired
+};
 export default Attributes;
+
+/**
+ * Check if component has multiplicity.
```

Comment:
I think it's fine in code context. At least it gives more context on top of the function name. Need to create core utils for these things anyways.